ui/app/registration/views/duplicatePatientTable.html

```diff
@@ -0,0 +1,42 @@
+<div class="registraition-search-results-container" ng-show="visibleData">
+    <h6>{{countNo}} {{::'REGISTRATION_LABEL_SIMILAR_PATIENT_FOUND' | translate}}</h6>
+        <table class="table">
+            <thead>
+            <tr>
+                <th><strong>{{ ::'REGISTRATION_LABEL_SYSTEM_ID' | translate}}</strong></th>
+                <th><strong>National ID</strong></th>
```

Comment:
@IipperMangesh `National ID` needs to be translated as well

Reply:
Yes, Now it is change as CNI.